classes/property_list.php

```diff
@@ -347,4 +350,17 @@ private function array_serialize_cftypes(array $array) : array {
         $array = new CFDictionary($array); // Convert back to CFDictionary so serialization is recursive.
         return $array->toArray(); // Serialize.
     }
+
+    /**
+     * Check if an array is associative or sequential.
+     *
+     * @param array $array Array to check.
+     * @return bool False if not associative.
+     */
+    private function is_associative_array(array $array) {
+        if (array() === $array) {
```

Comment:
What are you checking here?

Reply:
It just checks if array is empty. Stole the code. I'll fix it to be more readable